lib/adb.js

```diff
@@ -554,13 +570,35 @@ ADB.prototype.installApp = function installApp(deviceId, apkFile, opts, callback
 };
 
 /**
+ * Returns the ps output of the specified app and device/emulator, if running.
+ * @param {String} deviceId - The id of the device or emulator
+ * @param {ADB~psCallback} callback - A callback that is fired once ps is executed
+ */
+ADB.prototype.ps = function ps(deviceId, callback) {
+	var output = function (err, data) {
```

Comment:
`output` is a bad variable name since it is a function and not a buffer/string with "output".